packages/cra-template-seed/template/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## Description
```

Comment:
Not sure on the pull requests template to be honest.
While I think it's a good addition to a project setup, I'm not sure if we are going beyond our target here of providing an **application** seed.